llvm/test/CodeGen/PowerPC/is_fpclass.ll

```diff
@@ -117,10 +117,12 @@ define i1 @isinf_float(float %x) nounwind {
 define i1 @isinf_ppc_fp128(ppc_fp128 %x) nounwind {
 ; CHECK-LABEL: isinf_ppc_fp128:
 ; CHECK:       # %bb.0:
-; CHECK-NEXT:    xststdcdp 0, 1, 48
-; CHECK-NEXT:    li 3, 0
-; CHECK-NEXT:    li 4, 1
-; CHECK-NEXT:    iseleq 3, 4, 3
+; CHECK-NEXT:    addis 3, 2, .LCPI9_0@toc@ha
```

Comment:
This becomes worse for sure.

Reply:
Need special treatment for ppcf128: isfpclas is not legal for ppcf128 but legal for double, however, `isfpclass(ppcf128)` is actually lowered to `isfpclass(double)` (higher half).

Reply:
#105540 + #105577 should fix this